CODEOWNERS

```
@@ -122,6 +122,9 @@
 # renovate groupName=verify-source
 /task/verify-source @arewm @ralphbean
 
+# renovate groupName=ko-builder
+/task/ko-oci-ta @gallen @vdemeester @arewm @skoved @rh-hemartin @maruiz93
```

```suggestion
/task/ko-oci-ta @ggallen @vdemeester @arewm @skoved @rh-hemartin @maruiz93
```
Typo, this is someone else